### PR TITLE
Add signer password for file-based signers for Rekor

### PR DESCRIPTION
**Pull request by haydentherapper**
## Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [ ] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [ ] JSON Schema generated.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: charts/rekor/README.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ Part of the sigstore project, Rekor is a timestamping server and transparency lo
 | server.sharding.filename | string | `"sharding-config.yaml"` |  |
 | server.sharding.mountPath | string | `"/sharding"` |  |
 | server.signer | string | `"memory"` |  |
+| server.signerPassword | string | `""` |  |
```
Comment (k4leung4) [inline charts/rekor/README.md:172]: should we add a disclaimer this should only be used for testing?
passing password string here seems like bad practice in general

Comment (haydentherapper) [reply charts/rekor/README.md:172]: Another option is to omit this entirely, as extraArgs can cover this. WDYT?

Comment (k4leung4) [reply charts/rekor/README.md:172]: whats the motivation/use case for adding this arg?

extraArgs is also an option, especially it seems tedious to update the helm chart for every new flag

Comment (haydentherapper) [reply charts/rekor/README.md:172]: Just had someone accessing about signing options for Rekor and I noticed this was missing. Have we formed guidance on when we should add a top-level argument vs using extraArgs?

Comment (haydentherapper) [reply charts/rekor/README.md:172]: Thinking this over, I'm gonna close this out.

**Pull request closed without merging**